src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const entries = sourceString
```

Comment:
The variable `entries` is correctly initialized by splitting the `sourceString` using semicolons. Ensure that `sourceString` is always a valid string to avoid runtime errors.